src/Components/SmartComponents/Reports/ReportsPage.js

```diff
@@ -74,55 +88,66 @@ const ReportsPage = () => {
         return { string: tag, namespace, key, value };
     };
 
+    const fileIcon = <Icon size="lg">
+        <FileAltIcon color="var(--pf-v5-global--link--Color)" />
+    </Icon>;
+
+    const cards = [
+        {
+            key: 'executive-report-card',
+            title: EXECUTIVE_REPORT_CARD_TITLE,
+            icon: <ChartPieSolid style={styles.pieChartIcon} />,
+            body: EXECUTIVE_REPORT_CARD_DESCRIPTION,
+            footer: <DownloadExecutive />,
+            hasPermissions: true
+        },
+        {
+            key: 'report-by-cves-card',
+            title: CUSTOM_REPORT_CARD_TITLE,
+            icon: fileIcon,
+            body: CUSTOM_REPORT_CARD_DESCRIPTION,
+            footer: <a
+                className="create-report"
+                aria-label="Custom report button"
+                onClick={() => setModalOpen(true)}
+            >
+                {CUSTOM_REPORT_CARD_BUTTON}
+            </a>,
+            hasPermissions: canReadVulnerabilities
+        },
+        {
+            key: 'operating-system-report-card',
+            title: OPERATING_SYSTEM_REPORT_TITLE,
+            icon: fileIcon,
+            body: OPERATING_SYSTEM_REPORT_DESCRIPTION,
+            footer: <a
+                className="create-report"
+                aria-label="OS report button"
+                //onClick={() => setModalOpen(true)}
```

Comment:
Not sure if this was intentional or not since eventually this would be implement but just incase